aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -411,6 +412,24 @@ TORCH_META_FUNC(triangular_solve)(const Tensor& self, const Tensor& A, bool uppe
   }
 }
 
+TORCH_META_FUNC(_linalg_det)(const Tensor& A) {
+  at::native::squareCheckInputs(A, "linalg.det");
+  at::native::checkFloatingOrComplex(A, "linalg.det");
+
+  auto shape= A.sizes();
```

Comment:
Running `clang-format`, `git-clang-format`, or [`lintrunner`](https://github.com/pytorch/pytorch/wiki/lintrunner) would help with these formatting errors.
```suggestion
  auto shape = A.sizes();
```

Reply:
I thought this was done automatically? :/